Enterprise-CMCS/eAPD
# Remove Launchdarkly

Resolves the need for LaunchDarkly

### Description
Removes LaunchDarkly from the application. Turns all previously flagged code "on". Fully enables MMIS and adminCheck. Created a work around for the new code that shows or hides New Apdland. Created work around for sysadmin and just hardcoded access for the username 'sysadmin'.

### Chromatic Link
https://www.chromatic.com/pullrequest?appId=61d5b948cf6f17003a12bf77&number=4681

### Significant changes or possible side effects
Enabled all beta feature code.

### Steps to manually verify this change

1. Login as betauser and see that New Apdland is not available
2. Login as norole and see that New Apdland is available
3. Login as sysadmin and see that you have access to all states, click on one state
4. Verify that the Create new APD button isn't visible
5. Login as state admin, verify that you can create an MMIS APD and that it looks right

### This pull request is ready to code review when

      tests, refactor

### This pull request is ready to test when


### This pull request is ready to review when QA has


### This pull request can be merged when
